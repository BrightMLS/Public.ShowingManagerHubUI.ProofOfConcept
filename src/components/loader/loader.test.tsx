import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import Loader from './loader'

describe('<Loader /> Component', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<Loader visible={true} />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})

describe('<Loader /> Component', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<Loader visible={false} />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
