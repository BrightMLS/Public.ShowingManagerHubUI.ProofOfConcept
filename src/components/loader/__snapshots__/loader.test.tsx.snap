// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> Component should render without crashing 1`] = `
<Fragment>
  <WithStyles(ForwardRef(Backdrop))
    className="makeStyles-backdrop-4"
    open={true}
    transitionDuration={0}
  />
  <div
    className="makeStyles-spinnerContainer-1"
  >
    <div
      className="makeStyles-spinner-2"
    />
  </div>
</Fragment>
`;

exports[`<Loader /> Component should render without crashing 2`] = `""`;
